daemon/src/engine/framework/VirtualMachine.h

```diff
@@ -86,12 +86,17 @@ enum vmType_t {
 	TYPE_END
 };
 
+static std::unordered_map<std::string, int> vmCvarFlagsMap = {
```

Comment:
Here each file including VirtualMachine.h will have a new instance of that map, putting it in a cpp would be better.